pool/pool.go

```diff
@@ -245,6 +245,7 @@ type pool struct {
 	cancel     context.CancelFunc
 	closedCh   chan struct{}
 	cache      *SessionCache
+	stokenDur  uint64
```

Comment:
Maybe it'd be better to name the variable as `sTokenDuration` or `sessionTokenDuration` or something similar? Because at first sight it takes some time to get what `stoken` means here and type `uint64` doesn't  give more information.